pkgs/development/libraries/apr-util/default.nix

```diff
@@ -23,7 +23,7 @@ stdenv.mkDerivation rec {
 
   configureFlags = ''
     --with-apr=${apr} --with-expat=${expat}
-    --with-crypto
+    ${if !stdenv.isCygwin then "--with-crypto" else "--without-pgsql --without-sqlite2 --without-sqlite3 --without-freetds --without-berkeley-db --without-crypto"}
```

Comment:
Lots of negation in here! "if not cygwin then use crypto, otherwise (not not cygwin), I want no pgsql, no sqlite2/3, etc."